src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -165,7 +165,7 @@
                 <div class="col-md-10 col-sm-10 col-lr-pad
                   submit-upload-file">
                   <app-input (change)="validateInput($event.target.value)"
-                    [label]="'input_file'" [accept]="'.json, .zip, .txt, .tsv, .gz'"
+                    [label]="'input_file'" [accept]="'.json, .zip, .tsv, .gz, .txt'"
```

Comment:
@rlee7 was changing the order necessary? If not please revert this

Reply:
Hi, I reverted the unnecessary changes.